### PR TITLE
Updated error format for E0069

### LINKED ISSUE #35219: E0069 needs to be updated to new format
From: src/test/compile-fail/E0069.rs

Error E0069 needs a span_label, updating it from:

```
error[E0069]: `return;` in a function whose return type is not `()`
  --> src/test/compile-fail/E0069.rs:12:5
   |
12 |     return; //~ ERROR E0069
   |     ^^^^^^
```

To:

```
error[E0069]: `return;` in a function whose return type is not `()`
  --> src/test/compile-fail/E0069.rs:12:5
   |
12 |     return; //~ ERROR E0069
   |     ^^^^^^ return type is not ()
```

### PR DESCRIPTION
**Pull request by jaredwy**
Fixes #35219 which is part of the error bonanza at #35233

r? @jonathandturner

### CODE REVIEW

File: src/test/compile-fail/E0069.rs [hunk 1]
```diff
@@ -9,7 +9,10 @@
 // except according to those terms.
 
 fn foo() -> u8 {
-    return; //~ ERROR E0069
+    return;
+    //~^ ERROR `return;` in a function whose return type is not `()`
+    //~| NOTE return type is not ()
+
```
Comment (jonathandturner) [inline src/test/compile-fail/E0069.rs]: Thanks for PR - Looks good, but can you remove this extra line?

**Comment by jaredwy** [comment]
@jonathandturner let me know if you want me to rebase this.

**Comment by jonathandturner** [comment]
@jaredwy - would you mind squashing the commits?

**Comment by jonathandturner** [comment]
hehe, good call.  yes, please :)

**Comment by jaredwy** [comment]
@jonathandturner hopefully good to go now.

**Comment by jonathandturner** [comment]
Looks good!

@bors r+ rollup

**Pull request merged**